README.md

```diff
@@ -1,38 +1,15 @@
+Interactive map of functional connectivity measured by neural activtion in the nematode _C. elegans_. Live at [FunCoNN.princeton.edu](https://funconn.princeton.edu/).
 
-FunCoNN is built on top of [NemaNode](https://nemanode.org/) (NemaNode is reported in Witvliet et al., 2021) and adds Functional Connectivity data from the [Leifer Lab](http://leiferlab.princeton.edu) (citation pending) on top of the Chemical Synapse and Gap Junction data generated by the [Zhen](https://www.zhenlab.com), [Samuel](https://scholar.harvard.edu/aravisamuel), and [Lichtman](https://lichtmanlab.fas.harvard.edu) labs.
+FunCoNN is a fork of [NemaNode](https://nemanode.org/) (NemaNode is reported in Witvliet et al., 2021) that adds Functional Connectivity data from the [Leifer Lab](http://leiferlab.princeton.edu) on top of the Chemical Synapse and Gap Junction data generated by the [Zhen](https://www.zhenlab.com), [Samuel](https://scholar.harvard.edu/aravisamuel), and [Lichtman](https://lichtmanlab.fas.harvard.edu) labs.
+
+The f unctional connectivity data measured by neural activation is reported in: [Francesco Randi, Anuj K. Sharma, Sophie Dvali, Andrew M. Leifer, "A functional connectivity atlas of C. elegans measured by neural activation." arXiv:2208.04790 (2022)](https://doi.org/10.48550/arXiv.2208.04790).
 
 The project repositories, where you can peruse the code, request features, and report bugs are both located on GitHub:
 - [FunCoNN (BETA)](https://github.com/PrincetonUniversity/neuron-graph) (this project)
 - [NemaNode](https://github.com/zhenlab-ltri/NemaNode) (original project)
 
-## Required software
-- [Node.js](https://nodejs.org/en/) >= 11.15.0
-- [MySQL](https://www.mysql.com/downloads/) >= 5.6
-- [GCC/g++ compiler](https://packages.ubuntu.com/focal/build-essential)
-
-
-## Development setup
-
-1. Setup MySQL if first-time use: `sudo mysql_secure_installation`
-2. Install app dependencies by running `npm install`
-3. Setup MySQL user and database: `scripts/setup_config.sh funconn funconn_user password | sudo mysql`
-4. Populate database: `npm run populate-database`
-
-
-## Commands
-Once setup is completed, you will be able to run the following commands in this project directory:
-
-- `npm run watch` or `npm start`: start a app server to test alongside development (localhost:3000)
-- `npm run build-prod`: build the production version of the app
-- `npm run test`: run the tests
-- `populate-database`: take the raw data in src/server/db/raw-data and populate the mysql db with it
-
+This project is in beta and has not yet been peer reviewed. Data and code are released under a permissive license at https://github.com/PrincetonUniversity/neuron-graph/
 
-## Documentation
-For more information on how to do various things read the documentation
-- [Understanding the data used in funconn](docs/understanding-funconn-data.md)
-- [Adding new data to funconn](docs/adding-new-data.md)
-- [Deployment to AWS](docs/deployment-to-aws.md)
```

Comment:
Should we put this deleted documentation elsewhere? Perhaps in a `INSTALL.md` doc and/or a `CONTRIBUTING.md` doc?

Reply:
...and link those here perhaps?

Reply:
you can. but perhaps we can do that in a separate pull request commit as I think its loweor priority?

Reply:
Absolutely. #110